docs/_data/schema-meetups.yaml

```diff
@@ -0,0 +1,21 @@
+region: str(required=True)
+country: str(required=True)
+state: str(required=False)
+city: str(required=False)
+website: str(required=False)
+twitter: str(required=False)
+organizers: list(include('organizer-detail'), min=1, max=9)
+
+source: any(include('source_meetup'), include('source_other'), required=True)
+
+---
+
+organizer-detail:
+  name: str(required=True)
+  link: str(required=False, none=False)
+
+source_meetup:
+  meetup: str(required=True, none=False)
```

Comment:
Maybe we could call this and the other one "url"? So that you only flip whether url is under source_meetup or source_other. Feels cleaner.

Reply:
yeah, I actually went the other way to 

```
source:
  meetup:
```

and 

```
source:
   other:
```

but I'm falling into the trap of half assing things:

https://github.com/writethedocs/www/pull/2242#pullrequestreview-2373307871